app/views/users/index.html.slim

```diff
@@ -0,0 +1,23 @@
+h1 Listing users
+
+table
+  thead
+    tr
+      th Email
+      th Password
```

Comment:
are you really showing users passwords haha?

Reply:
> are you really showing users passwords haha?

apparently when the table is shown, only the email can be seen and the password is not shown at all